bootstrap/bootstrap.go

```diff
@@ -118,3 +129,19 @@ func SetHandlers() {
 	chassis.SetDefaultConsumerChains(consumerChainMap)
 	chassis.SetDefaultProviderChains(providerChainMap)
 }
+
+//InitEgressChain init the egress handler chain
+func InitEgressChain() error {
+	egresschain := strings.Join([]string{
+		handler.Router,
```

Comment:
is that necessary?

Reply:
Based on egress rule we need to decide which chain to select either egress chain or consumer chain. Egress chain need to be initalized which is done here.